trezorlib/tx_api.py

```diff
@@ -33,6 +33,17 @@ def get_url(self, resource, resourceid):
         url = '%s%s/%s' % (self.url, resource, resourceid)
         return url
 
+    def get_block(self, block_number):  # currunt block - block_number
```

Comment:
this whole function is highly questionable and I would prefer if it didn't exist, and you extracted the target block height from somewhere else

Reply:
as discussed on gitter: instead rename the function to, maybe, `bip115_get_nth_block`
and add a comment or docstring that explains that this gets the Nth block from top and returns its reversed hash and block height

Reply:
I would expect that `get_block(X)` call returns block with block_height==X, let's do it this way. Also the function does not return block, just it hash, so it should be called `get_blockhash(X)`.

To implement the intended behavior, let's create `get_blockheight()` function which returns the current blockheight and the desired outcome would be:

```
current = api.get_blockheight()
h = get_blockhash(current - 300)  # say in comment here why 300
```

Also not sure about `get_` prefix, maybe the functions should be called just `blockhash` and `blockheight`, but I leave that to @matejcik.

Reply:
yes, this is a good proposal. also the flipping should be done in client code, i.e., in `trezorctl`

as for naming, I don't particularly care, but my choice would be:
`current_height()` for the height function (it looks like a "property" of the API, which seems appropriate)
`get_block_hash(i)` for the other one